api/sheets/get_name.md

```diff
@@ -1,46 +1,65 @@
 ---
 layout: default
-title: Node Name
+title: Returning the Names Associated with a DCID
 nav_order: 1
 parent: Google Sheets
 grand_parent: API
 ---
 
-# Get a node's name
+# Returning the Names Associated with a DCID
 
-Given a cell or a column range of cells with [DCIDs](/glossary.html#dcid) (unique identifiers for all entities in the Data Commons knowledge graph), return the names associated with the given DCIDs.
+The `=DCGETNAME(*dcids*)` formula returns the names associated with given [DCIDs](/glossary.html#dcid) to a cell or a column range of cells.
 
-## General information about this formula
+Note: Be sure to follow the instructions for [Installing and Enabling the Sheets Add-On](https://docs.datacommons.org/api/sheets/) before using this formula.
 
-**Formula**: `=DCGETNAME(dcids)`
+## Formula
 
-**Required arguments**:
+```
+=DCGETNAME(dcids)
+```
 
-* `dcids` - A list of nodes to query, identified by their DCIDs.
+## Required Arguments
+
+`dcids` - A list of DCID nodes to query
 
```

Comment:
missing return values section before examples?

Reply:
I added it